Nodejs/Product/Nodejs/Debugger/ExceptionHandler.cs

```diff
@@ -84,117 +84,14 @@ sealed class ExceptionHandler {
-                "Error(EMLINK)",
-                "Error(EMSGSIZE)",
-                "Error(ENAMETOOLONG)",
-                "Error(ENETDOWN)",
-                "Error(ENETRESET)",
-                "Error(ENETUNREACH)",
-                "Error(ENFILE)",
-                "Error(ENOBUFS)",
-                "Error(ENODATA)",
-                "Error(ENODEV)",
-                "Error(ENOENT)",
-                "Error(ENOEXEC)",
-                "Error(ENOLINK)",
-                "Error(ENOLCK)",
-                "Error(ENOMEM)",
-                "Error(ENOMSG)",
-                "Error(ENOPROTOOPT)",
-                "Error(ENOSPC)",
-                "Error(ENOSR)",
-                "Error(ENOSTR)",
-                "Error(ENOSYS)",
-                "Error(ENOTCONN)",
-                "Error(ENOTDIR)",
-                "Error(ENOTEMPTY)",
-                "Error(ENOTSOCK)",
-                "Error(ENOTSUP)",
-                "Error(ENOTTY)",
-                "Error(ENXIO)",
-                "Error(EOVERFLOW)",
-                "Error(EPERM)",
-                "Error(EPIPE)",
-                "Error(EPROTO)",
-                "Error(EPROTONOSUPPORT)",
-                "Error(EPROTOTYPE)",
-                "Error(ERANGE)",
-                "Error(EROFS)",
-                "Error(ESPIPE)",
-                "Error(ESRCH)",
-                "Error(ETIME)",
-                "Error(ETIMEDOUT)",
-                "Error(ETXTBSY)",
-                "Error(EXDEV)",
-                "Error(MODULE_NOT_FOUND)",
-                "Error(SIGHUP)",
-                "Error(SIGINT)",
-                "Error(SIGILL)",
-                "Error(SIGABRT)",
-                "Error(SIGFPE)",
-                "Error(SIGKILL)",
-                "Error(SIGSEGV)",
-                "Error(SIGTERM)",
-                "Error(SIGBREAK)",
-                "Error(SIGWINCH)",
-                "EvalError",
-                "RangeError",
-                "ReferenceError",
-                "SyntaxError",
-                "TypeError",
-                "URIError",
-            };
-
             var defaultExceptionTreatments = new Dictionary<string, ExceptionHitTreatment>();
-#if FALSE
-            foreach (string exceptionType in exceptionTypes) {
-                defaultExceptionTreatments[exceptionType] = ExceptionHitTreatment.BreakAlways;
-            }
-#endif
 
-            foreach (string exceptionType in breakNeverTypes) {
-                defaultExceptionTreatments[exceptionType] = ExceptionHitTreatment.BreakNever;
+            // Get exception names from in NodePackage.Debugger.cs
+            foreach (var attr in System.Attribute.GetCustomAttributes(typeof(NodejsPackage))) {
+                var debugAttr = attr as ProvideNodeDebugExceptionAttribute;
+                if (debugAttr != null && !string.IsNullOrEmpty(debugAttr.ExceptionName)) {
+                    defaultExceptionTreatments[debugAttr.ExceptionName] = ExceptionHitTreatment.BreakNever;
+                }
```

Comment:
Rather than risk having this happen again, I refactored this code to remove the duplication. The core fix for the original problem was just the mismatch between using `Error()` one place and `Error` the other.